SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -16,6 +16,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 public:
 	USpatialGDKSettings(const FObjectInitializer& ObjectInitializer);
 
+#if WITH_EDITOR
+	virtual void PostEditChangeProperty(struct FPropertyChangedEvent& PropertyChangedEvent) override;
```

Comment:
Please don't pass parameters by non-const ref and preferably add forward declarations below includes.
In this case I don't think this needs forward declaring as the definition is already included.

Reply:
Just noticed it's virtual in which case we have to keep the dodgy non const :(. The declaration is still not needed I think.